### ISSUE TITLE
[fix] 	gql ingestion tests need to be updated to ingest the nouns first before the verbs

### ISSUE THREAD

**Issue by pxp928** [guacsec/guac#1000]
With the introduction of batch ingestion with #999, the [test data ingestion](https://github.com/guacsec/guac/blob/main/cmd/guacgql/cmd/ingest.go) needs to be updated to ingest `packages, sources, artifacts...etc` before calling `isDependency, IsOccurrence...etc`. Currently, it will output:

```
{"level":"error","ts":1688125602.265793,"caller":"cmd/ingest.go:88","msg":"Error in ingesting: input: certifyScorecard CertifyScorecard ::  input: Source type \"git\" not found\n\n","stacktrace":"github.com/guacsec/guac/cmd/guacgql/cmd.ingestScorecards\n\t/Users/parth/Documents/pxp928/artifact-ff/cmd/guacgql/cmd/ingest.go:88\ngithub.com/guacsec/guac/cmd/guacgql/cmd.ingestData\n\t/Users/parth/Documents/pxp928/artifact-ff/cmd/guacgql/cmd/ingest.go:44"}
{"level":"error","ts":1688125602.266811,"caller":"cmd/ingest.go:135","msg":"Error in ingesting: input: ingestSLSA IngestSLSA :: Subject artifact not found\n\n","stacktrace":"github.com/guacsec/guac/cmd/guacgql/cmd.ingestSLSA\n\t/Users/parth/Documents/pxp928/artifact-ff/cmd/guacgql/cmd/ingest.go:135\ngithub.com/guacsec/guac/cmd/guacgql/cmd.ingestData\n\t/Users/parth/Documents/pxp928/artifact-ff/cmd/guacgql/cmd/ingest.go:45"}
``` 
The solution will be similar to how the [unit tests in the inmemory backend](https://github.com/guacsec/guac/blob/main/pkg/assembler/backends/inmem/isOccurrence_test.go#L566-L589) are done.

**Comment by stevemenezes** [comment]
Can pick this issue up if the approach aligns with what is expected. For instances, for [this](https://github.com/guacsec/guac/blob/main/cmd/guacgql/cmd/ingest.go#L89) step, I just included the [ingest source](https://github.com/guacsec/guac/blob/main/pkg/assembler/clients/generated/operations.go#L23981) block, ran the ingestion locally and it gave no errors. Can you please confirm whether this is what is expected? Thanks.

**Comment by pxp928** [comment]
> Can pick this issue up if the approach aligns with what is expected. For instances, for [this](https://github.com/guacsec/guac/blob/main/cmd/guacgql/cmd/ingest.go#L89) step, I just included the [ingest source](https://github.com/guacsec/guac/blob/main/pkg/assembler/clients/generated/operations.go#L23981) block above it, ran the ingestion locally and it gave no errors. Can you please confirm whether this is what is expected? Thanks.

Yes that is correct!

**Comment by pxp928** [comment]
fixed with PR #1003
